server/mobs.cpp

```diff
@@ -291,6 +297,72 @@ Mob::~Mob()
 	}
 }
 
+struct MultiplayerMechanicsInfo {
```

Comment:
I think that "multiplayer mechanics" isn't really descriptive, maybe we could change it to "killstreak mechanics"? what do you think?

Reply:
The best I managed to come up with is "killcount mechanics", because that's basically what it is about, counting kills. If you're ok with the name, I'll push an update

Reply:
ok